From 973a5e7b00a66aa81c1d55719cb17db582c514fc Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 28 Oct 2021 13:46:56 +0000
Subject: [PATCH 432/462] Mantis 0061542: Move ThirdPartyLibraries to Globals

set DIR/INC/LIB-PAGESCAN/EPASS variables for global libraries

git-svn-id: svn://localhost/SelfServiceBuild/trunk@818 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntTargets.xml | 18 +++++-----
 SelfServiceBuild/Build/scripts/setenv.bat  | 42 ++++++++++++++++++++--
 2 files changed, 48 insertions(+), 12 deletions(-)

diff --git a/SelfServiceBuild/Build/java/AntTargets.xml b/SelfServiceBuild/Build/java/AntTargets.xml
index 7559e81..1174af2 100644
--- a/SelfServiceBuild/Build/java/AntTargets.xml
+++ b/SelfServiceBuild/Build/java/AntTargets.xml
@@ -159,7 +159,7 @@
     <condition property="javac.target" value="1.7">
         <not><isset property="javac.target"/></not>
     </condition>
-    
+
     <echo message="Compiler-Flags: debug=${javac.debug}"/>
     <echo message="Compiler-Flags: verbose=${javac.verbose}"/>
     <echo message="Compiler-Flags: deprecation=${javac.deprecation}"/>
@@ -290,7 +290,7 @@
            filesonly="${jar.filesonly}"
            manifest="${ManifestDir}/${ManifestName}">
       </jar>
-    </then>  
+    </then>
     </if>
 </target>
 
@@ -303,10 +303,10 @@
 
     <!-- Die erzeugten Unterverzeichnisse werden geloescht. -->
     <delete dir="${GenSrcDir}"/>
-    <delete dir="${GenXMLSrcDir}"/>    
+    <delete dir="${GenXMLSrcDir}"/>
     <delete dir="${ClassDir}"/>
-    <delete dir="${GenXMLClassDir}"/>    
-    <delete dir="${TestClassDir}"/>    
+    <delete dir="${GenXMLClassDir}"/>
+    <delete dir="${TestClassDir}"/>
     <delete dir="${ManifestDir}"/>
     <delete dir="${JavaDocDir}/${ant.project.name}"/>
 
@@ -362,7 +362,7 @@
           <path refid="testsource.classpath"/>
           <pathelement location="${JarName}"/>
           <pathelement location="${JarJunit}"/>
-        </classpath>           
+        </classpath>
         <src path="${TestSrcDir}"/>
     </javac>
 
@@ -376,10 +376,10 @@
     <junit printsummary="yes" haltonfailure="yes">
       <classpath>
         <path refid="test.classpath"/>
-        <pathelement location="${TestClassDir}"/>        
-        <pathelement location="${TestResourceDir}"/>        
+        <pathelement location="${TestClassDir}"/>
+        <pathelement location="${TestResourceDir}"/>
         <pathelement location="${JarName}"/>
-        <pathelement location="${JarJunit}"/>        
+        <pathelement location="${JarJunit}"/>
       </classpath>
 
 
diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index c3b386c..835cea5 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -10,7 +10,7 @@ set ERRORLEVEL=0
 if "%ENVIRONMENT_CALLED%_" == "1_" exit /b 0
 
 rem ######################################################################
-rem # MASSAIROOT 
+rem # MASSAIROOT
 rem # z.B.: C:\Viewstore\tpoersch_SelfServiceCommon_Main_integration
 rem ######################################################################
 
@@ -23,7 +23,7 @@ set ScriptDir=%ScriptDir:~0,-1%
 if NOT "%USE_PRIVATE_ENV%_" == "_" call setPrivateEnv.bat
 
 if defined MASSAIROOT goto massai_root_known
- 
+
 rem Das MASSAI-Verzeichnis liegt drei Ebenen darueber.
 set CurrentDir=%cd%
 cd %ScriptDir%\..\..\..
@@ -368,7 +368,7 @@ set SOAPCOMP=%DIR_GSOAP%\bin\soapcpp2.exe
 set DIR_GSOAP_2_7_17=%DIR_GLOBALSLIB%\gsoap\2.7.17\win32
 set WSDLCOMP_2_7_17=%DIR_GSOAP_2_7_17%\bin\wsdl2h.exe
 set SOAPCOMP_2_7_17=%DIR_GSOAP_2_7_17%\bin\soapcpp2.exe
-set SOAPCOMP_2_7_17_FLAGS=-I%DIR_GSOAP_2_7_17%\import 
+set SOAPCOMP_2_7_17_FLAGS=-I%DIR_GSOAP_2_7_17%\import
 
 set DIR_LUA=%DIR_GLOBALSLIB%\lua\5.0.a
 set INC_LUA=-I%DIR_LUA%\include
@@ -517,6 +517,42 @@ set INC_ZINT=-I%DIR_ZINT%\zint-2.6.3_rc2.src\backend
 set LIB_ZINT=%DIR_ZINT%\lib\zint.lib
 
 
+set DIR_PAGESCAN401=%DIR_GLOBALSLIB%\desko\PageScanApi\4.0.1
+set INC_PAGESCAN401=%DIR_PAGESCAN401%\include
+set LIB_PAGESCAN401=%DIR_PAGESCAN401%\lib\PageScanAPI.lib
+
+set DIR_PAGESCAN410=%DIR_GLOBALSLIB%\desko\PageScanApi\4.1.0
+set INC_PAGESCAN410=%DIR_PAGESCAN410%\include
+set LIB_PAGESCAN410=%DIR_PAGESCAN410%\lib\PageScanAPI.lib
+
+set DIR_PAGESCAN430=%DIR_GLOBALSLIB%\desko\PageScanApi\4.3.0
+set INC_PAGESCAN430=%DIR_PAGESCAN430%\include
+set LIB_PAGESCAN430=%DIR_PAGESCAN430%\bin\win32\PageScanAPI.lib
+
+set DIR_PAGESCAN460=%DIR_GLOBALSLIB%\desko\PageScanApi\4.6.0
+set INC_PAGESCAN460=%DIR_PAGESCAN460%\include
+set LIB_PAGESCAN460=%DIR_PAGESCAN460%\bin\win32\PageScanAPI.lib
+
+set DIR_PAGESCAN470=%DIR_GLOBALSLIB%\desko\PageScanApi\4.7.0
+set INC_PAGESCAN470=%DIR_PAGESCAN470%\include
+set LIB_PAGESCAN470=%DIR_PAGESCAN470%\bin\win32\PageScanAPI.lib
+
+set DIR_PAGESCAN481=%DIR_GLOBALSLIB%\desko\PageScanApi\4.8.1
+set INC_PAGESCAN481=%DIR_PAGESCAN481%\include
+set LIB_PAGESCAN481=%DIR_PAGESCAN481%\bin\win32\PageScanAPI.lib
+
+set DIR_EPASS1016=%DIR_GLOBALSLIB%\desko\EPassApi2\1.0.16
+set INC_EPASS1016=%DIR_EPASS1016%\include
+set LIB_EPASS1016=%DIR_EPASS1016%\lib\ePassApi2.lib
+
+set DIR_EPASS110=%DIR_GLOBALSLIB%\desko\EPassApi2\1.1.0
+set INC_EPASS110=%DIR_EPASS110%\include
+set LIB_EPASS110=%DIR_EPASS110%\lib\ePassApi2.lib
+
+set DIR_EPASS102014=%DIR_GLOBALSLIB%\desko\EPassApi2\102.0.14
+set INC_EPASS102014=%DIR_EPASS102014%\include
+set LIB_EPASS102014=%DIR_EPASS102014%\lib\win32\ePassApi2.lib
+
 rem ######################################################################
 rem # Externe Bibliotheken
 rem ######################################################################
-- 
2.41.0.windows.1

